libraries/ms-helper/inc/thermistor.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+// Thermistor Reading interface for NXRT15XH103FA1B (10k Ohm NTC)
+// Requires ADC, GPIO and Interrupts to be initialized.
+#include "adc.h"
+
+typedef struct {
+  uint32_t sibling_resistance;  // milliohms
+
+  ADCChannel adc_channel;
+} ThermistorStorage;
+
+typedef struct {
+  uint32_t sibling_resistance;  // milliohms
+
+  ADCChannel adc_channel;
+} ThermistorSettings;
```

Comment:
If you're only going to have 2 parameters there's no need for a struct.

Reply:
True, I can remove the structs